woocommerce/woocommerce-gateway-stripe
# Remove a unused confirmation modal and capability property

## Changes proposed in this Pull Request:

- Remove a modal to confirm that the merchant wants to disable the updated checkout experience.
This modal isn't included anywhere, thus removing it.
- Remove the `capability` property of the payment methods map on the JS side.
This property isn't used anywhere. Instead, we're building the capability string from the payment method ID when checking it. For example, in [#](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/develop/client/components/payment-method-capability-status-pill/index.js#L26), [#](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/develop/client/settings/account-activation-notice/index.js#L36), [#](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/develop/client/settings/general-settings-section/payment-methods-list.js#L191), [#](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/develop/client/settings/general-settings-section/payment-methods-unavailable-list.js#L14)

Not adding a changelog entry because it's a couple of small removals, but glad to include it if convenient.

## Testing instructions

Confirm the deleted modal and capability property is unused. The easiest way to do it is probably to run a search through the code.

---


**Post merge**
